nanshe_ipython.ipynb

```diff
@@ -1305,16 +1439,32 @@
         "\n",
         "\n",
         "if __IPYTHON__:\n",
-        "    result_image_stack = LazyZarrDataset(data_basename + postfix_f_f0 + zarr_ext, \"images\")\n",
-        "    lblimg = LazyZarrDataset(data_basename + postfix_rois + zarr_ext, \"labels\")\n",
+        "    fn = data_basename + postfix_f_f0 + zarr_ext\n",
+        "\n",
+        "    imgs_min, imgs_max = 0, 100\n",
+        "\n",
+        "    with open_zarr(fn, \"r\") as f:\n",
+        "        imgs = f[\"images\"]\n",
+        "        da_imgs = da.from_array(imgs, chunks=(norm_frames,) + imgs.shape[1:])\n",
```

Comment:
This should be using `block_frames` instead of `norm_frames` in this case. Fixing with PR ( https://github.com/nanshe-org/nanshe_workflow/pull/156 ).